packages/ui-avatar/src/Avatar/index.tsx

```diff
@@ -80,7 +80,7 @@ category: components
 @withStyle(generateStyle, generateComponentTheme)
 @testable()
 class Avatar extends Component<Props> {
-  static componentId = 'Avatar'
+  static componentId = 'Avatar' as const
```

Comment:
Why is this needed? Why not use `static readonly `?

Reply:
This was a mistake by me, I've made this change when I was exploring how to map the components to their generateTheme function, I've deleted it.